plaid/transactions.go

```diff
@@ -0,0 +1,103 @@
+package plaid
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type Transaction struct {
+	ID        string `json:"transaction_id"`
+	AccountID string `json:"account_id"`
+
+	Amount               float64  `json:"amount"`
+	Category             []string `json:"category"`
+	CategoryID           string   `json:"category_id"`
```

Comment:
there's also an `iso_currency_code` and `unofficial_currency_code` field

Reply:
updated!